gremlin-python/src/main/python/gremlin_python/driver/aiohttp/transport.py

```diff
@@ -16,29 +16,45 @@
 # specific language governing permissions and limitations
 # under the License.
 #
+from aiohttp import ClientResponseError
 import aiohttp
 import asyncio
-import async_timeout
-from aiohttp import ClientResponseError
+import sys
 
 from gremlin_python.driver.transport import AbstractBaseTransport
 
-__author__ = 'Lyndon Bauto (lyndonb@bitquilltech.com)'
+if sys.version_info >= (3, 11):
+    import asyncio as async_timeout
+else:
+    import async_timeout
```

Comment:
This and the `import sys` are the only change in this file, everything else came from `ruff`